nautobot_device_lifecycle_mgmt/filters.py

```diff
@@ -104,10 +106,10 @@ class SoftwareLCMFilterSet(NautobotFilterSet):
     q = django_filters.CharFilter(method="search", label="Search")
 
     device_platform = django_filters.ModelMultipleChoiceFilter(
-        field_name="device_platform__slug",
+        field_name="device_platform__model",
         queryset=Platform.objects.all(),
-        to_field_name="slug",
-        label="Device Platform (Slug)",
+        to_field_name="model",
```

Comment:
This should probably be changed to `name`. `Platform` doesn't have a `model` field.

Reply:
@gsnider2195  should field_name also be updated?  Just want to make sure this is covering the model change
```
   device_platform = django_filters.ModelMultipleChoiceFilter(
        field_name="device_platform__model",
        queryset=Platform.objects.all(),
        to_field_name="model",
        label="Device Platform (Model)",
    )
    ```

Reply:
Yes, also the label

Reply:
Effectively this?

```
    device_platform = django_filters.ModelMultipleChoiceFilter(
        field_name="device_platform__name",
        queryset=Platform.objects.all(),
        to_field_name="name",
        label="Device Platform (Name)",
    )
    ```